swmm-toolkit/test-requirements.txt

```diff
@@ -1,6 +1,6 @@
 
 
 
-pytest 
-numpy 
-aenum==3.0.0
+pytest==6.1.1
```

Comment:
if pytest and numpy are not a hard requirements, perhaps we should start use >= than ==. It would help us in the long term with newer versions of python.

Reply:
Sure I see ... in the dev branch that is true. But in the master branch where these changes are coming from everything needs to be pinned down when we release. Otherwise things will inevitably break if we ever need to go back to an older release.

Reply:
Sounds good, thanks @michaeltryby